### PR TITLE
Rel 813237 mark overwritten files

### PR DESCRIPTION
**Pull request by KamilKalitaRel**

### CODE REVIEW

**Review by KlaudiuszBryjaRelativity — DISMISSED** [review]

File: lib/TransferControllers/TransferControllerBase.cs [hunk 1]
```diff
@@ -423,6 +423,7 @@ protected virtual void Dispose(bool disposing)
                     else
                     {
                         this.TransferJob.Overwrite = true;
+                        this.TransferJob.IsOverwritten = true;
```
Comment (mmkowal-rel) [inline lib/TransferControllers/TransferControllerBase.cs:426]: I'm wondering if we could simplify this translation IsOverwritten -> TransferItemStatus. 
Maybe we can have this.TransferJob.State = Overwritten

File: lib/TransferJobs/SingleObjectTransfer.cs [hunk 2]
```diff
@@ -224,6 +224,7 @@ public override async Task ExecuteAsync(TransferScheduler scheduler, Cancellatio
                 if (TransferJobStatus.SkippedDueToShouldNotTransfer != this.transferJob.Status)
                 {
                     eventArgs.EndTime = DateTime.UtcNow;
+                    eventArgs.Status = this.transferJob.IsOverwritten ? TransferItemStatus.Overwritten : TransferItemStatus.Created;
```
Comment (mmkowal-rel) [inline lib/TransferJobs/SingleObjectTransfer.cs:227]: related to the previous comment

**Review by mmkowal-rel — APPROVED** [review]

File: lib/TransferControllers/TransferControllerBase.cs (hunk 1 above)
Comment (KamilKalitaRel) [reply lib/TransferControllers/TransferControllerBase.cs]: Done

File: lib/TransferJobs/SingleObjectTransfer.cs (hunk 2 above)
Comment (KamilKalitaRel) [reply lib/TransferJobs/SingleObjectTransfer.cs]: Done

**Review by mmkowal-rel — DISMISSED** [review]

File: version.txt [hunk 3]
```diff
@@ -1 +1 @@
-4.2.8
\ No newline at end of file
+4.2.9
```
Comment (KlaudiuszBryjaRelativity) [inline version.txt:1]: It looks that you don't push this version to code. @mmkowal-rel updated release procedure regarding how to bump version (via script or `.\build.ps1 Package` task). # #

**Review by KlaudiuszBryjaRelativity — CHANGES_REQUESTED** [review]

File: version.txt (hunk 3 above)
Comment (KamilKalitaRel) [reply version.txt:1]: Done

**Review by KlaudiuszBryjaRelativity — APPROVED** [review]

**Review by mmkowal-rel — APPROVED** [review]

**Pull request merged**